src/modules/service-catalog/components/service-catalog-list-item/SearchClearIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import ClearIcon from "@zendeskgarden/svg-icons/src/16/x-stroke.svg";
+import { Tooltip } from "@zendeskgarden/react-tooltips";
+import { IconButton } from "@zendeskgarden/react-buttons";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+
+const StyledIconButton = styled(IconButton)`
+  position: absolute;
+  top: 4px;
+  right: 4px;
```

Comment:
I just realized that this doesn't work for RTL languages. We should use `inset-inline-end` instead